createbillboardplaylist.py

```diff
@@ -311,6 +311,14 @@ def create_all(self):
             "http://www.billboard.com/charts/hot-100",
         )
 
+		# Billboard Country 50
```

Comment:
Two things on this line:

- According to [PEP8](https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces), which this project follows, we have to use spaces instead of tabs for indentation. You may have to change the settings in your editor to get it to insert spaces instead of tabs. You can run `pylint createbillboardplaylist.py` and `flake8 createbillboardplaylist.py` to check the coding style.

- The chart name should be "Hot Country Songs"

```suggestion
		# Billboard Hot Country Songs
```